docs/pom.xml

```diff
@@ -110,17 +66,17 @@
 					<plugin>
 						<groupId>org.asciidoctor</groupId>
 						<artifactId>asciidoctor-maven-plugin</artifactId>
-						<configuration>
-							<attributes>
-								<project-version>${project.version}</project-version>
-							</attributes>
-						</configuration>
 					</plugin>
 					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
 						<artifactId>maven-antrun-plugin</artifactId>
 					</plugin>
 					<plugin>
 						<artifactId>maven-deploy-plugin</artifactId>
+						<goals>
+							<goal>deploy</goal>
```

Comment:
Why are you adding these goals, even though we didn't need them before?

Reply:
I was copy/pasting from https://github.com/spring-cloud/spring-cloud-build#reusing-the-documentation, but you're right, this should be skipped instead.